samples/visual-indicator/nodejs/pages/configure.html

```diff
@@ -0,0 +1,30 @@
+<head>
+    <script src="https://res.cdn.office.net/teams-js/2.11.0/js/MicrosoftTeams.min.js"
+            integrity="sha384yBjE++eHeBPzIg+IKl9OHFqMbSdrzY2S/LW3qeitc5vqXewEYRWegByWzBN/chRh" crossorigin="anonymous">
+    </script>
+</head>
+
+<body>
+    <div class="font-semibold font-title" style="color:gray">Configure your app here.</div>
+    <h2 style="color:dodgerblue;font-family:sans-serif">
+        Visual indicator for your Teams app
```

Comment:
Show visual indicator in Teams Meetings app

Reply:
fixed as per comment